pele_platform/plotter.py

```diff
@@ -0,0 +1,1374 @@
+    group.add_argument('-c', '--csv_file', type=str, default=None,
+                       help='Path to PELE csv file')
+    group.add_argument('-r', '--results_folder', type=str,
+                       default=None, help='Path to results folder')
+    group.add_argument('-o', '--output_folder', type=str,
+                       default=None, help='Path to PELE output folder')
+    parser.add_argument('--report_name', type=str, default='report',
+                        help='Name of PELE\'s report files. ' +
+                        'Only required when extracting data from ' +
+                        'PELE\'s output folder')
+    parser.add_argument('--trajectory_name', type=str,
+                        default='trajectory.pdb',
+                        help='Name of PELE\'s trajectory files. ' +
+                        'Only required when extracting data from ' +
+                        'PELE\'s output folder')
+    parser.add_argument('-t', '--plot_type', type=str,
+                        choices=PLOT_TYPE_CHOICES, default=DEFAULT_PLOT_TYPE,
+                        help='Plot type. One of ' + str(PLOT_TYPE_CHOICES))
+    parser.add_argument("-x", "--xaxis", metavar=('INT', 'STR'),
+                        type=str, nargs='*',
+                        help="Column number to plot on the X axis. "
+                        + "Optionally, the axis label can be defined "
+                        + "afterwards", default=None)
+    parser.add_argument("-y", "--yaxis",  metavar=('INT', 'STR'),
+                        type=str, nargs='*',
+                        help="Column number to plot on the Y axis. "
+                        + "Optionally, the axis label can be defined "
+                        + "afterwards", default=None)
+    parser.add_argument("-z", "--zaxis",  metavar=('INT', 'STR'),
+                        type=str, nargs='*',
+                        help="Column number to plot on the Y axis. "
+                        + "Optionally, the axis label can be defined "
+                        + "afterwards", default=None)
+    parser.add_argument('--xlowest', type=float, default=None,
+                        help='Lowest value to plot on the X axis')
+    parser.add_argument('--xhighest', type=float, default=None,
+                        help='Highest value to plot on the X axis')
+    parser.add_argument('--ylowest', type=float, default=None,
+                        help='Lowest value to plot on the Y axis')
+    parser.add_argument('--yhighest', type=float, default=None,
+                        help='Highest value to plot on the Y axis')
+    parser.add_argument('--zlowest', type=float, default=None,
+                        help='Lowest value to plot on the Z axis')
+    parser.add_argument('--zhighest', type=float, default=None,
+                        help='Highest value to plot on the Z axis')
+    parser.add_argument('--colormap', type=str, default='gnuplot',
```

Comment:
Add a list of available colour maps to command line help.

Reply:
It is going to be already in the documentation and the list of arguments is already too large. For now I would not add them there.